packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -74,6 +73,18 @@ protected function setUp(): void
         $bootstrapFilesIncluder->includeBootstrapFiles();
     }
 
+    protected function tearDown(): void
+    {
+        unset(
+            $this->applicationFileProcessor,
+            $this->parameterProvider,
+            $this->dynamicSourceLocatorProvider,
+            $this->removedAndAddedFilesCollector,
+            $this->originalTempFileInfo,
+        );
+        gc_collect_cycles();
```

Comment:
Why this is needed? Some comment would be helpful to avoid removing in the future after running test in Linux with no effect